test/e2e/lib/helpers/session.ts

```diff
@@ -1,16 +1,25 @@
-import { SESSION_STORE_KEY } from '@datadog/browser-core'
-import { deleteAllCookies } from './browser'
+import { SESSION_STORE_KEY, SESSION_TIME_OUT_DELAY } from '@datadog/browser-core'
+import {
+  getInitialSessionState,
+  toSessionState,
+  toSessionString,
+} from '@datadog/browser-core/src/domain/session/sessionState'
```

Comment:
**💭 thought:** ‏Types and constants are fine, but e2e tests should not rely on library functions.